packages/salesforcedx-vscode-core/src/conflict/conflictNode.ts

```diff
@@ -43,7 +51,24 @@ export class ConflictNode extends vscode.TreeItem {
   }
 
   get tooltip() {
-    return this._conflict ? this._conflict.relPath : this.label;
+    if (this._conflict) {
+      let tooltipMessage: string = '';
+      if (this._conflict.remoteLastModifiedDate) {
+        tooltipMessage += nls.localize(
+          'conflict_detect_remote_last_modified_date',
+          `${new Date(this._conflict.remoteLastModifiedDate).toLocaleString()}`
+        );
+      }
+      if (this._conflict.localLastModifiedDate) {
+        tooltipMessage += nls.localize(
+          'conflict_detect_local_last_modified_date',
+          `${new Date(this._conflict.localLastModifiedDate).toLocaleString()}`
```

Comment:
@violetyao Is this the format we landed on after the last conversation during parking lot?

Reply:
It's based on our slack discussion in [this thread](https://platformcloud.slack.com/archives/GCDT2EVAQ/p1624298023397700)! I'll add this item to today's parking lot to double check the format.